pkgs/by-name/gi/gitlab-container-registry/package.nix

```diff
@@ -19,6 +19,11 @@ buildGoModule rec {
 
   vendorHash = "sha256-aKE/yr2Sh+4yw4TmpaVF84rJOI6cjs0DKY326+aXO1o=";
 
+  env = {
+    # required for go test
+    AZURE_DRIVER_VERSION = "azure_v2";
+  };
+
   postPatch = ''
     # Disable flaky inmemory storage driver test
     rm registry/storage/driver/inmemory/driver_test.go
```

Comment:
I believe this can be reactivated, the test has since then been rewritten and I didn't notice failures on my machine without nix builds (was removed a while back when this test would sometimes fail in nix build).

Reply:
I'd suggest doing a follow up PR once we have working release branches again.